### ISSUE TITLE
Why is Issus analis blocked?

### ISSUE THREAD

**Issue by mdoering** [CatalogueOfLife/data#503]
seems like a real species:
https://pubmed.ncbi.nlm.nih.gov/28183130/
https://species.wikimedia.org/wiki/Issus

blocked: https://www.checklistbank.org/catalogue/3/decision?limit=100&offset=0&subjectDatasetKey=1011

**Comment by yroskov** [comment]
We are not able to get data from DBTNT databases (including FLOW) after switch to the CLB. FLOW update in June 2020 was experimental and unsuccessful. Ideally, it should not appear in production CoL. 

Name **Issus analis Brullé, 1833** listed 7 times as CoL synonym to Zopherisca analis (Brullé, 1833) in FLOW dataset at CLB. Six entries should be blocked, one should left as a synonym and go in CoL. Blocking decision for all 7 entries is a mistake

@Geoff, if we cannot update FLOW properly, would it be possible to get back FLOW@ac19 dataset?

**Comment by mdoering** [comment]
we still use the ac19 FLOW dataset and have it as an ACEF source from the AC19 release: https://github.com/CatalogueOfLife/data/blob/master/ACEF/11.tar.gz

**Comment by mdoering** [comment]
There is no Issus analis in the last COL releases...

**Comment by yroskov** [comment]
>There is no Issus analis in the last COL releases...

Yes, all 7 entries of Issus analis as a synonym of Zopherisca analis were blocked in FLOW@CLB of June 2020. It was a mistake. One entry should be unblocked.

The best solution would be to bring back FLOW@ac19 until we get a modern update from FLOW.

**Comment by gdower** [comment]
@yroskov, the FLOW AC19 dataset is imported.

**Comment by yroskov** [comment]
Thanks! I'll do next steps today

**Comment by yroskov** [comment]
@gdower, could you pls give me a link to the "new" FLOW in CLB?

Species statistics in FLOW in AC19 differ with this results:  https://www.checklistbank.org/dataset/1011/imports

13466 spp vs 13151 spp in ac19 http://www.catalogueoflife.org/annual-checklist/2019/details/database/id/11

**Comment by yroskov** [comment]
OK. At least, _Issus analis_ is there and present only once: https://www.checklistbank.org/dataset/1011/taxon/21523

**Comment by gdower** [comment]
It was the proxy dataset version before, not the AC19 archive.

**Comment by yroskov** [comment]
Do we have actual metadata with the proxy version (date of release)?

**Comment by yroskov** [comment]
FLOW from ac19 re-synced 2023-01026
